apis/Google.Cloud.Logging.V2/docs/index.md

```diff
@@ -19,6 +19,8 @@ package](https://www.nuget.org/packages/Logary.Targets.Stackdriver)
 and follow the [instructions on the project
 homepage](https://github.com/logary/logary#stackdriver-target-alpha-level).
 
+[Serilog](https://serilog.net/) has a sink output to Google Stackdriver Logging. Install [Nuget package](https://www.nuget.org/packages/Serilog.Sinks.GoogleCloudLogging/) `Serilog.Sinks.GoogleCloudLogging` and [configure options](https://github.com/manigandham/serilog-sinks-googlecloudlogging).
```

Comment:
Nit: please fix "Nuget" to "NuGet", and ideally add "the" after "Install"

Reply:
All set!